diesel/src/query_source/aliasing/macros.rs

```diff
@@ -84,6 +84,10 @@ macro_rules! alias {
                 pub const $alias_name: Alias<$alias_ty> = $($table)::+ as $alias_name;
             )*
         }
+        $(
+            #[allow(non_camel_case_types)]
+            type $alias_name = $crate::query_source::Alias::<$alias_ty>;
```

Comment:
Took the option here to not put the implicit type alias in the explicit `const` syntax.
I'd like a second opinion on this. (I have no strong perference as I'm not using this syntax anyway.)

Reply:
I think I would prefer to have it always there, just to have the always the same generated code. That makes it probably easier to work with that kind of code in the context of macros.